cognite_toolkit/_cdf_tk/utils/fileio/_readers.py

```diff
@@ -49,8 +52,32 @@ def from_filepath(cls, filepath: Path) -> "FileReader":
         )
 
 
+class NDJsonReader(FileReader):
+    format = ".ndjson"
```

Comment:
"jsonl" should also be supported, right?

Reply:
One format at at time. I could add support for that as well, but for now (to keep the PRs small) I limit it.